### DISCUSSION TITLE
[Desktop] How to create a slash command like the Notion

### DISCUSSION THREAD

**Discussion by Joahyan** [singerdmx/flutter-quill#888] [Desktop]
![image](https://user-images.githubusercontent.com/40434066/180389356-392bb8fa-6dfd-4986-93c6-f4d7f96b250b.png)

**Comment by singerdmx** [comment]
Not familiar with that.
Is it like a special key?

**Comment by singerdmx** [comment]
I think it can be done like `mention`
It can be achieved via link https://github.com/singerdmx/flutter-quill/discussions/637
